analytics/client.go

```diff
@@ -29,33 +30,15 @@ type clientImpl struct {
 
 	// TODO: Remove Mixpanel once we've confirmed that Segment is working.
 	mixpanelClient mixpanel.Mixpanel
+
+	// The internal client used to send events to Amplitude.
+	amplitudeClient amplitude.Client
+	// App info from which event is sent
+	amplitudeAppInfo amplitude.EventOptions
```

Comment:
This comment is hard to understand. I have a suggestion for improvement below. However, `amplitudeClient` really belong together `amplitudeAppInfo`, so we'd be better served by having an abstraction that wraps them both and injects `amplitudeAppInfo` into all reported tracking events. **Non-blocking.**

```suggestion
	// This is included in all tracking events reported to Amplitude.
	amplitudeAppInfo amplitude.EventOptions
```